test/integration/global_deploy_test.rb

```diff
@@ -116,4 +126,54 @@ def test_global_deploy_task_failure
       "Template validation failed",
     ])
   end
+
+  def test_global_deploy_prune_black_box_success
+    assert_deploy_success(deploy_global_fixtures('globals', clean_up: false, selector: 'test=prune1'))
```

Comment:
same question, should we be adding this selector to the generated one so we don't have to hope future contributors won't reuse this manual selector?